build.gradle

```diff
@@ -334,8 +334,8 @@ dependencies {
     implementation group: 'com.github.hmcts', name: 'document-management-client', version: '7.0.0'
     implementation group: 'com.github.hmcts', name: 'cmc-pdf-service-client', version: '7.0.1'
 
-    implementation group: 'com.github.hmcts', name: 'sscs-common', version: '5.5.0'
-    implementation group: 'com.github.hmcts', name: 'sscs-pdf-email-common', version: '5.2.5'
+    implementation group: 'com.github.hmcts', name: 'sscs-common', version: 'f0be329'
+    implementation group: 'com.github.hmcts', name: 'sscs-pdf-email-common', version: '569da4c'
```

Comment:
Is this commit outdated? Should be [2a7b3ca](https://github.com/hmcts/sscs-pdf-email-common/pull/1111/commits/2a7b3ca406b8ff86742e60b829ada0b20c0257c2)

Reply:
that's the suppression commit so don't need to use it